fitnessllm_dataplatform/batch_handler.py

```
@@ -118,39 +138,36 @@ def process_user(
             )
             refresh_function = REFRESH_FUNCTION_MAPPING[data_source.value]
-            strava_data = self.get_user_stream_data(
-                uid=user_id, data_source=data_source
-            )
-            refresh_function(self.db, user_id, strava_data["refreshToken"])
+            strava_data = self.get_user_stream_data(uid=uid, data_source=data_source)
+            refresh_function(self.db, uid, strava_data["refreshToken"])
 
-            self.startup.full_etl(uid=user_id, data_source=data_source.value)
+            process_user = ProcessUser(uid=uid, data_source=data_source.value)
+            process_user.full_etl(uid=uid, data_source=data_source.value)
```

Pylint (3.3.7)

[error] Unexpected keyword argument 'uid' in method call

(E1123)